docs/guides/best-practices.md

```diff
@@ -6,12 +6,16 @@ While it's easy to get started building a Pack, there are lots of options to exp
 ## Formulas
 
 - [x] Follow the guidance on [formula naming][formulas_naming].
-- [x] Ensure each formulas has a few [examples defined][formulas_examples].
+- [x] Ensure each formula has a few [examples defined][formulas_examples]. These show up in the generated documentation for the formula.
+- [x] Set a [suggested value][parameter_suggested_value] for required parameters. This allows users to drag the formula from the side panel and see a result immediately.
+- [x] For each formula that has only a single required parameter, consider exposing it as a [column format][column_formats]. This makes it easier for users to use the functionality in a table.
+- [x] Consider the [caching behavior][formulas_caching] of your formulas. Use longer cache times for formulas where the results are deterministic, and shorter cache times for ones that fetch external data.
 
 
 ## Actions
 
 - [x] Follow the guidance on [action naming][actions_naming].
+- [x] Ensure each action formula [returns a result][actions_result]. This allows the doc to react to the action being completed.
```

Comment:
It might be worth elaborating on this, e.g. "For an action with no meaningful value to return, simply returning a non-empty value like 'OK' by convention is useful and appropriate."

Reply:
This is already included in the section being linked to:

> In some cases there is no meaningful result, in which case the convention is to return the string "OK".